### PR TITLE
bugfix: delete empty ip string which will be added to api server cert alt name

### PR DESCRIPTION
**Pull request by kakaZhou719**
### Describe what this PR does / why we need it

1. `make` will add default value  "" as one element , while  empty string is meaningful for k8s api server alt name cert.
thus, it will fail to run `kubeadm init`.
2. skip validate masters ip if only scale up node.

### CODE REVIEW

File: types/api/v2/cluster_types.go [hunk 1]
```diff
@@ -75,13 +75,13 @@ func (in *Cluster) GetMasterIPList() []net.IP {
 	return in.GetIPSByRole(common.MASTER)
 }
 
-func (in *Cluster) GetMasterIPStrList() []string {
+func (in *Cluster) GetMasterIPStrList() (ipStrList []string) {
```
Comment (allencloud) [inline types/api/v2/cluster_types.go:78]: I think the modification is useless. The effect of original and modified code are the same. @kakaZhou719

Comment (kakaZhou719) [reply types/api/v2/cluster_types.go:78]: make will add default value "" to ipStrList, after element  append, the result looks like:  `["","172.16.11.8"]`

Comment (allencloud) [reply types/api/v2/cluster_types.go:78]: No, I do not think make will add default value "" to ipStrList.  A unit test case is the best way to show the truth. @kakaZhou719

**Review by allencloud — APPROVED** [review]

**Comment by allencloud** [comment]
LGTM

**Pull request merged**